src/sbvr-api/permissions.ts

```
@@ -684,6 +684,161 @@ const getAlias = (name: string) => {
+
+						const collapsedPermissionFilters = buildODataPermission(
+							permissionsLookup,
+							methodPermissions.GET,
+							vocabulary,
+							targetResourceEscaped,
+							odata,
+						);
+
+						_.set(
+							odata,
+							['tree', 'options', '$filter'],
+							collapsedPermissionFilters,
+						);
+
+						const canAccessFunction: ResourceFunction = function(
+							property: AnyObject,
+						) {
+							// remove method property so that we won't loop back here again at this point
+							delete property.method;
+
+							if (!this.defaultResource) {
+								throw new Error(`No resource selected in AST.`);
+							}
+
+							const targetResourceAST = this.NavigateResources(
+								this.defaultResource,
+								property.name,
+							);
+
+							const targetResourceName = sqlNameToODataName(
+								targetResourceAST.resource.name,
+							);
+							const currentResourceName = sqlNameToODataName(
+								this.defaultResource.name,
+							);
+
+							if (
+								currentResourceName === targetResourceEscaped &&
+								targetResourceName === escapedName
+							) {
+								foundCanAccessLink = true;
+							}
+							// return a true expression to not select the relationship, which might be virtual
+							// this should be a boolean expression, but needs to be a subquery in case it
+							// is wrapped in an `or` or `and`
+							return [
+								'Exists',
+								['SelectQuery', ['Select', [['Alias', ['Number', 1], 'id']]]],
+							];
+						};
+
+						// We need execute the
+						const odata2AbstractSQL = new OData2AbstractSQL(
+							originalAbstractSQLModel,
+							{
+								canAccess: canAccessFunction,
+							},
+						);
+
+						odata2AbstractSQL.match(odata.tree, 'GET', [], odata.binds.length);
+
+						if (foundCanAccessLink) {
+							arr[1][0] = `${arr[1][0]}$bypass`;
+						}
+					} catch (e) {
+						if (e === constrainedPermissionError) {
+							// ignore
+							return;
+						}
+					}
```

This currently ignores all errors, I expect you want:
```suggestion
					} catch (e) {
						if (e === constrainedPermissionError) {
							// ignore
							return;
						}
						throw e;
					}
```